### PR TITLE
MM-T4275 added e2e tests for board description

### PR DESCRIPTION
**Pull request by adj2908**
#### Summary
This PR adds cypress e2e test for setting up Board description. 

#### Ticket Link
This ticket is from focalboard repo. 
[Github issue link](https://github.com/mattermost/focalboard/issues/1813
)
#### Release Note
```None```

### CODE REVIEW

**Comment by saturninoabril** [comment]
The `Upload to S3 bucket` is known CI issue and not related to your changes. I'll review your PR, thanks @adj2908!

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @adj2908! I ran multiple tests and I find it flaky. It's not consistently passing. Could you help confirm it it's not the case for you? And if any, find ways to make it consistently passed.

File: e2e/cypress/tests/integration/boards/create_board_spec.ts [hunk 1]
```diff
@@ -47,6 +47,54 @@ describe('Create and delete board / card', () => {
             should('have.value', 'Testing');
     });
 
+    it('MM-T4275 Set up Board description', () => {
+        cy.visit('/boards');
+
+        // # Create empty board and change tile to Testing
```
Comment (furqanmlk) [inline e2e/cypress/tests/integration/boards/create_board_spec.ts:53]:
```suggestion
        // # Create an empty board and change tile to Testing
```

**Review by furqanmlk — APPROVED** [review]
@adj2908,  LGTM, just a minor change. I am approving it.

File: e2e/cypress/tests/integration/boards/create_board_spec.ts (hunk 1 above)
Comment (adj2908) [reply e2e/cypress/tests/integration/boards/create_board_spec.ts]: Thanks for the approval. I have made the changes suggested by you.

**Comment by adj2908** [comment]
@saturninoabril Can you please rerun the checks as it was failing last time due to lint errors? I have fixed the errors now.

File: e2e-tests/cypress/tests/integration/boards/create_board_spec.ts [hunk 2]
```diff
@@ -47,6 +47,54 @@ describe('Create and delete board / card', () => {
...
     });
 
+    it('MM-T4275 Set up Board description', () => {
+        cy.visit('/boards');
+
+        // # Create an empty board and change tile to Testing
+        cy.findByText('Create an empty board').should('exist').click({force: true});
+        cy.get('.BoardComponent').should('exist');
+
+        // Change Title
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/boards/create_board_spec.ts:57]:
```suggestion
        // # Change Title
```

File: e2e-tests/cypress/tests/integration/boards/create_board_spec.ts [hunk 3]
```diff
@@ -47,6 +47,54 @@ describe('Create and delete board / card', () => {
...
+        // # Click on "Add a description" below the board title and type "for testing purposes only"
+        cy.findByText('Add a description...').should('be.visible').wait(timeouts.HALF_SEC);
+
+        // * Assert that the editable description should be visible
+        cy.findByText('Add a description...').should('be.visible');
+        cy.findByText('Add a description...').click({force: true});
+        cy.get('.description').
+            click().
+            get('.description .MarkdownEditorInput').
+            type('for testing purposes only');
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/boards/create_board_spec.ts:80]: I'm still getting some flakiness in test with `for testing purposes only` not getting set as the description. A workaround I found is to click anywhere to give trigger and time for description to be saved. Let me know if it works for you.

```suggestion
            type('for testing purposes only');

        // # Click to other element to give some time for the description to be saved.
        cy.findByPlaceholderText('Untitled board').click();
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @adj2908! Couple of suggestions then I'm good.

**Comment by adj2908** [comment]
@saturninoabril I have made changes based on the feedback you gave. Please review it. Thanks.

**Comment by adj2908** [comment]
@saturninoabril Looks like you will have to manually merge the PR. Can you please check?

**Pull request merged**